specs-go/config.go

```diff
@@ -320,6 +320,16 @@ type LinuxNetwork struct {
 	Priorities []LinuxInterfacePriority `json:"priorities,omitempty"`
 }
 
+// LinuxRdma for Linux cgroup 'rdma' resource management (Linux 4.11)
+type LinuxRdma struct {
+	// Hca device name whose resources to be restricted
+	HcaDevice string `json:"hca_device"`
+	// Maximum number of HCA handles that can be opened. Default is "no limit".
+	HcaHandles uint32 `json:"hca_handles"`
+	// Maximum number of HCA objects that can be created. Default is "no limit".
+	HcaObjects uint32 `json:"hca_objects"`
```

Comment:
Are zero values allowed (e.g. for “you can't use any HCA handles”)?  If not, we should say so in the Markdown spec.  If so, we should make these `omitempty` pointers so we can distinguish between “set to zero” and “unset”.

Reply:
yes. setting zero is allowed.